packages/math-input/src/components/keypad/index.tsx

```diff
@@ -2,41 +2,34 @@ import Color from "@khanacademy/wonder-blocks-color";
 import {View} from "@khanacademy/wonder-blocks-core";
 import {StyleSheet} from "aphrodite";
 import * as React from "react";
+import {useState} from "react";
```

Comment:
Could we rather use `React.useState()` instead of re-importing symbols from 'react'?

Reply:
I'll change it, but I think we need to have a longer discussion about this because this.

``` JSX
// This is what I've always done
import React, {useState} from "react"

function MyComponent() {
  const [something, setSomething = useState()
  return <div />
}
```

TypeScript seems to require the `*` which makes the import more annoying.


``` JSX
// What I would prefer
import * as React from "react"
import {useState} from "react"

function MyComponent() {
  const [something, setSomething = useState()
  return <div />
}

// or

// What you would prefer
import * as React from "react"

function MyComponent() {
  const [something, setSomething = React.useState()
  return <div />
}
```

React is on v18 and we're on v16 (from 2020). In v17 they dropped the requirement to import React.

``` JSX
// No more React import
import {useState} from "react"

function MyComponent() {
  const [something, setSomething = useState()
  return <div />
}
```

So my argument would be:
1. We should upgrade React to v18 before we accumulate so much tech debt that we need a FE Goliath.
2. We can then remove the need for the React import.
3. At that point we will be doing `const [something, setSomething = useState()` which I think is the community norm.

If we did things the way I prefer, the updates to files post-upgrade would require a 1 line change (removing `import * as React from "react"`). If we did things the way you prefer, we would need to do 1+n updates (where n is each time we used a hook; to remove the `React.` part).

Anyway, I don't see either option as wrong and I think this is a larger discussion for FE devs overall, so I'm just going to change it.